src/getting-started/build-a-sample-app.md

```diff
@@ -70,22 +75,51 @@ impl Component for Model {
     }
 }
 
-fn main() {
-    yew::initialize();
+#[wasm_bindgen(start)]
+pub fn run_app() {
     App::<Model>::new().mount_to_body();
 }
 ```
 {% endcode %}
 
 This template sets up your root `Component`, called `Model` which shows a button that updates itself when you click it. Take special note of `App::<Model>::new().mount_to_body()` inside `main()` which starts your app and mounts it to the page's `<body>` tag. If you would like to start your application with any dynamic properties, you can instead use `App::<Model>::new().mount_to_body_with_props(..)`.
 
+Finally, add an `index.html` file under a new `static` folder for your app.
```

Comment:
I think it should be 
Finally, add an `index.html` file ~~under~~ into a new  folder called `static`  ~~folder~~ ~~for~~ in your app.